src/models/methods/baseMethod.ts

```diff
@@ -1,4 +1,5 @@
 export interface BaseRequest {
+  [x: string]: unknown;
```

Comment:
Not sure if this is the right way to solve the problem, but this fixes TypeScript's complaint in `client.request` that `account` doesn't exist on all `Request` objects (even though we don't need it to)